core/authentication_api.php

```diff
@@ -409,6 +416,8 @@ function auth_auto_create_user( $p_username, $p_password ) {
 
 	if( $t_login_method == BASIC_AUTH ) {
 		$t_auto_create = true;
+    } else if ( $t_login_method == HTTP_AUTH && isset( $_SERVER['REMOTE_USER'] ) ) {
```

Comment:
Is the `isset()` check necessary?

Reply:
Yes, since original HTTP_AUTH execution path is not guaranteed to has a valid 'REMOTE_USER' variable which is required to rightful web-server authentication.